examples/dolly-pythia-fine-tuned/create_data_with_dolly_and_pythia.ipynb

```diff
@@ -41,33 +41,31 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "openai.api_key = \"your key here\"\n",
-    "assert openai.api_key != \"your key here\", \"Set your key\""
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from dataclasses import replace\n",
     "import datetime\n",
     "import locale\n",
-    "import re\n",
     "import time\n",
     "import uuid\n",
     "\n",
-    "from datasets import load_dataset\n",
     "import openai\n",
     "import pandas as pd\n",
     "import phoenix as px\n",
     "import torch\n",
-    "from transformers import AutoTokenizer\n",
+    "from datasets import load_dataset\n",
+    "from transformers import AutoModelForCausalLM, AutoTokenizer\n",
     "\n",
     "locale.getpreferredencoding = lambda: \"UTF-8\"  # This resolves a Colab bug that occurs sometimes."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "openai.api_key = \"your key here\"\n",
+    "assert openai.api_key != \"your key here\", \"Set your key\""
+   ]
+  },
```

Comment:
if adding this let's use getpass instead.

Reply:
This isn't added, I just moved a cell around to fix a linting issue